### PR TITLE
banana-cake-pop: Add version 1.0.0-preview.3

### PR DESCRIPTION
**Pull request by TheRandomLabs**
Banana Cake Pop is a GraphQL IDE which works well with Hot Chocolate and any other GraphQL server.

### CODE REVIEW

File: bucket/banana-cake-pop.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            "hash": "537f0ca66954456cb924450e8a0919b238b3fe9f8c0cdd6540b199f8e0af2935"
+        }
+    },
+    "installer": {
+        "script": [
+            "Move-Item \"$dir\\`$PLUGINSDIR\\app-64.7z\" -Destination $dir",
+            "Expand-7ZipArchive \"$dir\\app-64.7z\" -Removal",
+            "Remove-Item -Recurse \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall Banana Cake Pop.exe\""
+        ]
+    },
```
Comment (Ash258) [inline bucket/banana-cake-pop.json:18]:
```suggestion
```

File: bucket/banana-cake-pop.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "1.0.0-preview.3",
+    "description": "GraphQL IDE",
+    "homepage": "https://hotchocolate.io/docs/banana-cakepop",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://blob.chillicream.io/banana/BananaCakePop-1.0.0-preview.3.exe#/dl.7z",
+            "hash": "537f0ca66954456cb924450e8a0919b238b3fe9f8c0cdd6540b199f8e0af2935"
```
Comment (Ash258) [inline bucket/banana-cake-pop.json:9]:
```suggestion
            "hash": "537f0ca66954456cb924450e8a0919b238b3fe9f8c0cdd6540b199f8e0af2935",
            "pre_install": [
                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" -DestinationPath \"$dir\"",
                "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninst*\" -Recurse"
            ]
```

File: bucket/banana-cake-pop.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            "Expand-7ZipArchive \"$dir\\app-64.7z\" -Removal",
+            "Remove-Item -Recurse \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall Banana Cake Pop.exe\""
+        ]
+    },
+    "bin": [
+        [
+            "Banana Cake Pop.exe",
+            "bananacakepop"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/banana-cake-pop.json:24]: There is no command line support

```suggestion
```

File: bucket/banana-cake-pop.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    "shortcuts": [
+        [
+            "Banana Cake Pop.exe",
+            "Banana Cake Pop"
+        ]
+    ],
+    "checkver": {
+        "regex": "BananaCakePop-([\\w.-]+)\\.exe"
+    },
```
Comment (Ash258) [inline bucket/banana-cake-pop.json:33]:
```suggestion
    "checkver": "BananaCakePop-([\\w.-]+)\\.exe",
```

File: bucket/banana-cake-pop.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://blob.chillicream.io/banana/BananaCakePop-1.0.0-preview.3.exe#/dl.7z"
```
Comment (Ash258) [inline bucket/banana-cake-pop.json:37]:
```suggestion
                "url": "https://blob.chillicream.io/banana/BananaCakePop-$version.exe#/dl.7z"
```

**Comment by alanlima** [comment]
Thanks @TheRandomLabs 
I love your changes.

**Pull request merged**